dynamodb/src/main/java/site/ycsb/db/DynamoDBClient.java

```diff
@@ -35,7 +37,7 @@
 import java.util.Vector;
 
 /**
- * DynamoDB v1.10.48 client for YCSB.
+ * DynamoDB v1.10.48 client for YCSB. Updated to use v 1.11.812.
```

Comment:
just say it's using the v 1.11.812 or that folks should look at the pom file for the specific client version